mycroft/tts/google_tts.py

```diff
@@ -32,6 +33,12 @@ def get_tts(self, sentence, wav_file):
         Returns:
             Tuple ((str) written file, None)
         """
+        langs = tts_langs()
+        if self.lang.lower() not in langs:
+            if self.lang[:2].lower() in langs:
+                self.lang = self.lang[:2]
+            else:
+                raise ValueError("Language not supported by gTTS: %s" % lang)
```

Comment:
This should be `self.lang` right?

Also we try to use the `.format()` for string formatting so can you change it to
`raise ValueError('Language not supported by gTTS: {}'.format(self.lang)`

Reply:
just done.